components/printful/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@pipedream/awork",
+  "version": "0.0.2",
+  "description": "Pipedream Awork Components",
+  "main": "awork.app.mjs",
```

Comment:
Please change to: "printful.app.mjs"

Reply:
This is a differente Printful, `printful` app is for `Api Key` authentication!